README.md

```diff
@@ -4,6 +4,116 @@ CoreTools Healthcheck: [![CoreTools healthcheck](https://azure-functions.visuals
 
 # Dockerhub
 
+## V4 Images
+
+#### Dotnet
+
+`mcr.microsoft.com/azure-functions/dotnet`
+
+Linux amd64 Tags
+
+| Tags                                       | Dockerfile                                                              | OS Version |
+|--------------------------------------------|-------------------------------------------------------------------------|------------|
+| `4`                                      | [Dockerfile](host/4/bullseye/amd64/dotnet/dotnet-inproc/dotnet.Dockerfile)            | Debian 10  |
+| `4-slim`                                 | [Dockerfile](host/4/bullseye/amd64/dotnet/dotnet-inproc/dotnet-slim.Dockerfile)       | Debian 10  |
+| `4-appservice`, `4-dotnet3-appservice` | [Dockerfile](host/4/bullseye/amd64/dotnet/dotnet-inproc/dotnet-appservice.Dockerfile) | Debian 10  |
+| `4-dotnet3-core-tools` | [Dockerfile](host/4/bullseye/amd64/dotnet/dotnet-inproc/dotnet-core-tools.Dockerfile) | Debian 10  |
+
+
+`mcr.microsoft.com/azure-functions/dotnet-isolated`
+
+| Tags                                       | Dockerfile                                                              | OS Version |
+|--------------------------------------------|-------------------------------------------------------------------------|------------|
+| `4`                                      | [Dockerfile](host/4/bullseye/amd64/dotnet/dotnet-isolated/dotnet-isolated.Dockerfile)            | Debian 10  |
```

Comment:
I think this points to the old docker file (which we should delete from the repo to avoid confusion @CooperLink ). The actual docker file will be generated in the `out` directory when we run the `generate-composite.sh` script (example path: `\host\4\bullseye\amd64\out\dotnet\dotnet-isolated-appservice.Dockerfile`)

Reply:
This brings back the question of whether we should check-in the generated files (the `out` directory) to the source control. I feel like it might be a good idea to do so. If we want to check something quickly on the dockerfile we are using (ex: what version of extension bundle/aspnet sdk we are including in the final docker image produced).  Thoughts @CooperLink @pragnagopa

Reply:
+1. In the last few weeks some customers were also looking for the final dockerfiles for reference (not just to use as a base image) to build their completely custom dockerfiles and having the final artifact directly available would definitely help.

Reply:
Final output docker files will be part of the release artifacts. Each release will have attached  assets - waiting for a release to ensure this flow works. For now, will remove the files from the table above ?

Reply:
DockerHub docs updated - https://hub.docker.com/_/microsoft-azure-functions-dotnet - could one of you approve this PR. I will have a follow up PR once we start generating docker files as part of next release.